### PR TITLE
Add Nanomodem messages

### PR DESCRIPTION
**Pull request by natkokrasevac**

### CODE REVIEW

File: msg/NanomodemPayload.msg [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# types of messages
+uint8 BRDCST = 0
+uint8 UNICST = 1
+
+# raw message array
+uint8[] msg
+
+# id of the sender (0 - 255)
+# -1 if unknown
+int16 sender_id
```
Comment (dulanad) [inline msg/NanomodemPayload.msg]: Either int8 or int32.

File: msg/NanomodemRequest.msg [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+uint8 req_type
+
+# scheduled or unscheduled message TRUE or FALSE
+uint8 scheduled
+
+# raw message array
+uint8[] msg
+
+# request id (0 - 255)
+uint8 id
```
Comment (dulanad) [inline msg/NanomodemRequest.msg]: Which one will it be, uint8, int8, uint32, int32?

**Comment by dulanad** [comment]
I would make this more generic to cover a wider range of acoustic modems. A good start would be to adjust the naming of the messages. Add nicer comment formatting for the fields. In other pull requests we can then adjust the fields when the current format is tested with more acoustic devices.

**Comment by natkokrasevac** [comment]
These messages are currently in labust_msgs repository. Since we are open-sourcing the simulator soon, we thought it would be best to remove labust_msgs dependency from grpc_ros_adapter. 
I agree with your remarks, I will check if the id field type changes require any changes on the grpc_ros_adapter side and fix it.

For the names of the messages, I think maybe we change it later so that we don't slow down open-sourcing even more

**Pull request merged**